### PR TITLE
Update skeletor to work with v1.0.0-alpha.22

### PR DESCRIPTION
**Pull request by carolynvs**
* Use v1.0.0-alpha.22 of Porter
* Switch from using the old magefile helper in Porter to using the new magefiles repository
* Add the TestPublish magefile target
* Rename get.porter.sh/porter/pkg/context to portercontext
* Change the skeletor go mod name so that it's easier for people to find/replace their way to a custom mixin. Using our vanity domain wasn't really helping anyone.
* Fix bad reference to packr, skeletor uses go:embed now.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,4 +1,4 @@
-module get.porter.sh/mixin/skeletor
+module github.com/getporter/skeletor
```
[the remark this answers is not in the document]
Comment (carolynvs) [reply go.mod:1]: I am naming it "plugin-template" for the plugins. I would like to rename skeletor to mixin-template to match. Skeletor is a reference to an old 80's cartoon (he man) and that wasn't a great idea for a project with users from around the world.

[the remark this answers is not in the document]
Comment (VinozzZ) [reply go.mod:1]: Wow, after some googling, turns our he man and she ra is in the same universe! I loved the she-ra remake!

**Review by VinozzZ — APPROVED** [review]
I assume we want to do the rename in another PR

File: go.mod (hunk 1 above)
[the remark this answers is not in the document]
Comment (carolynvs) [reply go.mod:1]: The new she-ra on netflix is great! 🌈 ✨

**Comment by carolynvs** [comment]
Yes, I will do the rename separately. 👍

**Pull request merged**